disnake/interactions/base.py

```diff
@@ -871,6 +871,9 @@ async def edit_message(
         a component interaction or modal interaction (if the modal was sent in
         response to a component interaction).
 
+        .. versionchanged:: 2.5
+            Now supports editing the original message of modal interactions.
```

Comment:
Needs a newline in between the directive and the body, otherwise it ends up on the same line. 
![image](https://user-images.githubusercontent.com/71233171/157559375-386d8edc-33c2-4f27-920d-d0fc72a86713.png)

Reply:
Changed it, but either it's still cached or the newline doesn't affect the spacing.